.github/dependabot.yml

```
@@ -401,3 +401,28 @@ updates:
       - "dependencies"
       - "pip"
       - "terraform"
+  # Tezosqa
+  - package-ecosystem: "docker"
+    directory: "/tezosqa/0.1"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "dependencies"
+      - "pip"
```

`package-ecosystem: "docker"` so:
```suggestion
      - "docker"
```